tox.ini

```diff
@@ -21,6 +21,7 @@ setenv =
     VIRTUALENV_DOWNLOAD = 0
 deps =
     pytest-timeout
+    ruamel.yaml>=0.15
```

Comment:
Shouldn't this be already installed by the plugin itself?

Reply:
I got an error about missing `ruamel.yaml` when running tox. Might be something broken with my setup though. I'll investigate.

Reply:
Seems like it isn't needed. I deleted my .tox directory and tried again without the dependency in tox.ini and it worked.